swap_meet/vendor.py

```diff
@@ -1,2 +1,207 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        '''
+        Constructor of Vendor class:
+        Parameters:
+            inventory - represents all of the objects of the classs Item
+              that self has and is willing to swap with other vendors
+        '''
+
+        #If no argument is passed for inventory it will initialize as an empty list
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+
+    def add(self, new_item):
+
+        self.inventory.append(new_item)
+        return new_item
+    
+    def remove(self, old_item):
+        '''
+        input:
+            self -
+            old_item - object from the Item class that self will remove from its inventory list 
+        output:
+            old_item - if old_item is found in the inventory list it will be removed and returned 
+                side effect- inventory will have the old_item removed 
+            False - if old_item is not found inventory will not change
+        '''
+
+        if old_item in self.inventory:
+
+            self.inventory.remove(old_item)
+            return old_item
+
+        else:
+            return False
+        
+    
+    def get_by_id(self, items_id):
+        '''
+        Finds the item in self.inventory with the id = items_id
+        '''
+
+        for item in self.inventory:
+            if items_id == item.id: 
+                return item
```

Comment:
The Wave 2 requirements ask you to *explicitly* return `None`. When you don't put a return statement, you're *implicitly* returning `None`.

>If there is no matching item in the inventory, the method should explicitly return None

```python
    def get_by_id(self, item_id):
        for item in self.inventory:
            if item_id == item.id:
                return item
        return None
```